tests/test_rp_vars.py

```diff
@@ -29,15 +29,15 @@ def test_get_and_set_rp_vars(new_solver_session_no_transcript) -> None:
 def test_get_all_rp_vars(new_solver_session_no_transcript) -> None:
     case_path = download_file("Static_Mixer_main.cas.h5", "pyfluent/static_mixer")
     solver = new_solver_session_no_transcript
-    solver.file.read(file_type="case", file_name=case_path)
+    solver.tui.file.read_case(case_path)
     rp_vars = solver.rp_vars
     # all vars
     all_vars = rp_vars()
     assert len(all_vars) == pytest.approx(9000, 10)
 
     # refresh
-    solver.file.write(file_type="case", file_name=case_path)
-    solver.file.read(file_type="case", file_name=case_path)
+    solver.tui.file.write_case(case_path)
+    solver.tui.file.read_case(case_path)
```

Comment:
I'm not sure if this is related to the test that is currently failing, but is it a good idea to use the `tui` here?